gitpod-network-check/cmd/checks.go

```diff
@@ -198,13 +209,22 @@ func checkSMPrerequisites(ctx context.Context, ec2Client *ec2.Client) error {
 		}
 
 		if len(response.VpcEndpoints) == 0 {
-			if endpoint.Required {
-				return fmt.Errorf("❌ VPC endpoint %s not configured: %w", endpoint.Endpoint, err)
+			if strings.Contains(endpoint.Endpoint, "execute-api") {
```

Comment:
:bubbles: Instead of doing a substring match on the name, would it make sense to mark the endpoint with a flag named e.g. "testInMainSubnet" or similar...? :thinking:

Reply:
> would it make sense to mark the endpoint with a flag named e.g. "testInMainSubnet" or similar...? 🤔

At the moment, I think no. This is for a couple reasons:

1. I don't expect the service name to change, and we're not checking for the service name in other places. If we were, then I'd consider adding a flag.
2. This section of tests is for asserting we can interact with EC2 and SSM services, execute-api service has no impact on CLI functionality. It does help us assert private DNS is enabled, but, only when the VPC endpoint exists in the same account as Gitpod.